datadog_lambda/tracing.py

```diff
@@ -19,17 +20,48 @@
 )
 from ddtrace import tracer, patch
 from ddtrace import __version__ as ddtrace_version
+from ddtrace.filters import TraceFilter
 from ddtrace.propagation.http import HTTPPropagator
 from datadog_lambda import __version__ as datadog_lambda_version
 
 logger = logging.getLogger(__name__)
 
+
+SPAN_TYPE_TAG = "_dd.span_type"
+SPAN_TYPE_INFERRED = "inferred"
+
+
+class InferredSpanFilter(TraceFilter):
```

Comment:
I think this class definitely deserve a comprehensive docstring to explain why it's needed and the background.